src/windowsTerminal.ts

```diff
@@ -163,7 +168,7 @@ export class WindowsTerminal extends Terminal {
     });
   }
 
-  private _defer(deferredFn: Function): void {
+  private _defer(deferredFn: Function, arg?: any): void {
```

Comment:
Is the arg part used at all? Should we just do `.call(this)`?

Reply:
Yes its needed for the deferred writeas it needs to inject the data chunk.